web/src/channel_folders.ts

```diff
@@ -17,6 +17,9 @@ export function init(): void {
 
 init();
 
+export const MAX_CHANNEL_FOLDER_NAME_LENGTH = 100;
+export const MAX_CHANNEL_FOLDER_DESCRIPTION_LENGTH = 1024;
```

Comment:
Server does not inlcude these fields in `/register` response currently. Do we want to include them there and avoid maintaining the value at two different places?